ols/app/models/config.py

```diff
@@ -265,6 +316,16 @@ def __eq__(self, other):
 
     def validate_yaml(self) -> None:
         """Validate conversation cache config."""
+        if self.type is not None:
```

Comment:
if type is none isn't that also an error?

Reply:
come to think of it, it is

Reply:
So `conversation_cache` is optional, but if it's there, it has to have a type and the config for the type.

Reply:
we should make it required.  you can pick the in-memory one or redis, but you need to configure one.